src/course-unit/utils.test.ts

```diff
@@ -0,0 +1,23 @@
+import { adoptCourseSectionUrl } from './utils';
+
+describe('adoptCourseSectionUrl', () => {
+  it('should transform container URL correctly', () => {
+    const params = {
+      url: '/container/block-v1:edX+DemoX+Demo_Course+type@vertical+block@vertical_0270f6de40fc',
```

Comment:
[question/optional]: Should we get this value from the data we use for mocks? This applies to all data in this file that we can take from mocks.

Reply:
Simplified test cases